ainblockchain/docs
# [docs] Clean quick start

## Summary
- install latest version sdk
- add public RPC endpoint and event handler endpoint
- add how to get native ain
- remove transfer(step5) and nonce(step6) section
- remove duplicate code

To-do
- check echo bot status
- upload whole quick start code as repo
